command/rootchain/params.go

```diff
@@ -0,0 +1,206 @@
+package rootchain
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/rootchain"
+	"github.com/umbracle/ethgo/abi"
+)
+
+const (
+	dirFlag                = "dir"
+	rootchainAddrFlag      = "rootchain-addr"
+	eventABIFlag           = "event-abi"
+	methodABIFlag          = "method-abi"
+	localAddrFlag          = "local-addr"
+	payloadTypeFlag        = "payload-type"
+	blockConfirmationsFlag = "block-confirmations"
+)
+
+var (
+	params = &rootchainParams{}
+)
+
+const (
+	defaultRootchainName = "rootchain.json"
```

Comment:
Does it make sense to put this in .gitignore as we did for other json files

Reply:
Added 👍 

[ad5dcff](https://github.com/0xPolygon/polygon-edge/pull/716/commits/ad5dcff42668045b0d03607e7f0399885b7b4d38)